server/sandbox/sandbox.go

```diff
@@ -1,4 +1,4 @@
-package server
+package sandbox
```

Comment:
@mrunalp Probably need to be a subpackage there, too (if we move kpod state out into a subpackage, we'll end up with another import cycle). `libkpod/sandbox` would be fine with me.